integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/ConversationService.java

```diff
@@ -65,6 +66,7 @@ public Conversation registerStatus(Conversation conversation, MessageStatus stat
                     !conversation.isMsh()) {
                 conversation.setPollable(true);
             }
+            System.out.println("SAVING!!!!!!!!!!!!!!!!!!!!!!");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS106)